pdr_backend/sim/sim_engine.py

```diff
@@ -57,7 +57,9 @@ def __init__(
         self.exchange = exchange_mgr.exchange(
             "mock" if mock else ppss.predictoor_ss.exchange_str,
         )
-
+        self.position_open = ""  # long, short, ""
```

Comment:
(Putting here arbitrarily: please git merge main, to avoid discrepancy with main branch. Just a few changes but it threw me off.)